providers/openfeature-provider-flagd/pyproject.toml

```diff
@@ -69,3 +69,8 @@ omit = [
   "src/openfeature/contrib/provider/flagd/proto/*",
   "tests/**",
 ]
+
+[tool.pytest.ini_options]
+addopts = [
+    "--import-mode=importlib",
+]
```

Comment:
What is this setting for?

Reply:
Was to prevent errors with test files with same name in different folders, but it is not needed and according to docs it isn't clearly preferred over the default, so we can remove this: https://docs.pytest.org/en/7.1.x/explanation/pythonpath.html